_data/simple-icons.json

```diff
@@ -15901,6 +15901,11 @@
             "hex": "333333",
             "source": "https://github.com/unocss/unocss/blob/fc2ed5bb6019b45565ff5293d4b650522f1b79b4/playground/public/icon.svg"
         },
+        {
+            "title": "unpkg",
+            "hex": "000000",
```

Comment:
```suggestion
            "hex": "242424",
```

I'm seeing `#242424` used as the background of their [GitHub](https://github.com/unpkg) and [Twitter](https://twitter.com/unpkg) profile pictures.

![](https://github.com/unpkg.png)

Reply:
Good catch! I'm still not sure about this color. There is no activity on Twitter since 2018 and no activity on this [Github](https://github.com/unpkg) since 2022. On their [website](https://unpkg.com/), there is no usage of this greyish color, but they use black a lot (favicon, text, footer).

I'm not even sure what this Github org is, since unpkg is hosted on Github at https://github.com/mjackson/unpkg 🤔

Reply:
Their favicon uses black so I would vote for keeping it black: https://raw.githubusercontent.com/mjackson/unpkg/master/public/favicon.ico